src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -116,11 +114,8 @@
             handleInstancesValue: function (instances) {
                 this.selectedInstances = instances
             },
-            handleChangeLogTypeValue: function (type: string) {
-                this.changeLogTypeValue = type
-            },
-            handleChangeLogMessageValue: function (message: string) {
-                this.changeLogMessageValue = message
+            handleChangelog: function (changelog: object | null) {
```

Comment:
is there a need for the type here ?

Reply:
Yeah good point. It's not strictly necessary because we don't have types for this component, so it's just documentation really. But hopefully useful nevertheless.

However... we should have types here, which I'll do in a maintenance ticket - which would fix your concern?

Reply:
Would be nice if you add a link to the maintenance ticket here...

Reply:
Thanks... mrc-2560